app/Routes.hs

```diff
@@ -25,7 +28,7 @@ routes staticDir aboutContents privacyContents = [
     ("course-info", look "dept" >>= courseInfo),
     ("depts", liftIO deptList),
     ("timesearch", searchResponse),
-    ("calendar", lookCookieValue "selected-lectures" >>= calendarResponse),
+    -- ("calendar", lookCookieValue "selected-lectures" >>= calendarResponse),
```

Comment:
Restore this route, and rename your "calendar" route back to "timetable-image". I guess you should add a new link to the export modal.